### PR TITLE
Preview Hotkeys work consistently

### LINKED ISSUE #1724: Preview Modal Next/Previous keyboard hotkeys only work once
**Describe the bug**
The preview navigation hotkeys (Left/Right arrow) only work once. Subsequent attempts to use these do no result in the next/previous file being previewed

**To Reproduce**
1. Log in, navigate to a folder with multiple (more than 2) files
2. Preview the first file
3. Push the right arrow key on the keyboard
4. Preview the second file
5. Push the right arrow key on the keyboard
6. The second file is still active in the preview

**Expected behavior**
The 3rd file should be previewed, and pushing the left arrow key should preview the 2nd file

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: Brave
 - Version: 1.31.91

**Additional context**

### PR DESCRIPTION
**Pull request by FSM1**
Closes #1724

### CODE REVIEW

**Review by asnaith — APPROVED** [review]
Working nicely :) 

I wasn't aware that we could navigate with keys until now 😅. I've added it to the list of user scenarios.

**Comment by FSM1** [comment]
> Working nicely :)
> 
> I wasn't aware that we could navigate with keys until now 😅. I've added it to the list of user scenarios.

`Escape` also closes the preview. 

We should add some automated tests for this functionality.
```
cy.get('body').type(
  '{leftarrow}{rightarrow}'
)
```

**Comment by asnaith** [comment]
> We should add some automated tests for this functionality.

Yeah @FSM1 👍. Seems like a good candidate for it and most of the page object stuff for preview is already there. I've created an issue for it #1750

File: packages/files-ui/src/Components/Modules/FilePreviewModal.tsx [hunk 1]
```diff
@@ -241,11 +241,11 @@ const FilePreviewModal = ({ file, nextFile, previousFile, closePreview, filePath
 
   useHotkeys("Left,ArrowLeft", () => {
     previousFile && previousFile()
-  })
+  }, [previousFile])
 
   useHotkeys("Right,ArrowRight", () => {
     nextFile && nextFile()
-  })
+  }, [nextFile])
```
Comment (RyRy79261) [inline packages/files-ui/src/Components/Modules/FilePreviewModal.tsx:248]: ffs 😂

**Review by RyRy79261 — APPROVED** [review]

File: packages/files-ui/src/Components/Modules/FilePreviewModal.tsx (hunk 1 above)
Comment (FSM1) [reply packages/files-ui/src/Components/Modules/FilePreviewModal.tsx:248]: That was pretty much my attitude as well 😂

**Pull request merged**